AK/Utf16View.cpp

```diff
@@ -370,11 +384,11 @@ u32 Utf16CodePointIterator::operator*() const
     //    W2 as its 10 low-order bits.
     // 5) Add 0x10000 to U' to obtain the character value U. Terminate.
 
-    auto code_unit = host_code_unit(*m_ptr, Endianness::Host);
+    auto code_unit = *m_ptr;
 
     if (Utf16View::is_high_surrogate(code_unit)) {
         if (m_remaining_code_units > 1) {
-            auto next_code_unit = host_code_unit(*(m_ptr + 1), Endianness::Host);
+            auto next_code_unit = *(m_ptr + 1);
```

Comment:
why is it correct to yeet the endianness wrapper here?

Reply:
`host_code_unit(..., Host)` is a noop by definition.